test/evergreen.yml

```diff
@@ -5167,9 +5167,6 @@ buildvariants:
     CMAKE_PREFIX_PATH: -DCMAKE_PREFIX_PATH="$(pwd)/TCMALLOC_LIB"
     python_binary: '/opt/mongodbtoolchain/v4/bin/python3'
   tasks:
-  # FIXME-WT-8947: For now run the whole stress test suite like we do on the x86 platform. But later
```

Comment:
We have decided to [keep all stress tests enabled on ARM](https://jira.mongodb.org/browse/WT-8947?focusedCommentId=4548857&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4548857)